vnext/Desktop.ABITests/React.Windows.Desktop.ABITests.vcxproj

```diff
@@ -61,6 +61,7 @@
       <ForcedIncludeFiles>pch.h</ForcedIncludeFiles>
       <SDLCheck>true</SDLCheck>
       <AdditionalIncludeDirectories>$(MSBuildThisFileDirectory);$(VCInstallDir)UnitTest\include;$(ReactNativeDir)\ReactCommon;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <DisableSpecificWarnings>4646;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
fyi that we are working on actually removing the "Disable warnings" by fixing the warnings

Reply:
We don't have this anymore ..